src/shared/lib/StoreUtils.spec.ts

```diff
@@ -498,4 +499,16 @@ describe('StoreUtils', () => {
                 "fetchStudy should be called with the correct study id (study4)");
         });
     });
+
+    describe('wholeGenomeProfiled()', () => {
+        it('handles undefined values', () => {
+            assert.isTrue(noGenePanelUsed(undefined));
+        });
+        it('handles "WES"-tag values', () => {
+            assert.isTrue(noGenePanelUsed('WES'));
+        });
+        it('handles "WGS"-tag values', () => {
+            assert.isTrue(noGenePanelUsed('WGS'));
+        });
```

Comment:
can you add a test where it returns false?

Reply:
Added!